src/main/kotlin/de/bund/digitalservice/useid/config/SecurityConfig.kt

```diff
@@ -26,14 +25,11 @@ class SecurityConfig(
     @Bean
     fun securityFilterChain(http: HttpSecurity): SecurityFilterChain {
         return http.authorizeHttpRequests()
-            .requestMatchers("$IDENTIFICATIONS_OLD_BASE_PATH/*/tc-token").permitAll()
-            .requestMatchers("$IDENTIFICATIONS_OLD_BASE_PATH/*/tokens/*").permitAll()
-            .requestMatchers("$IDENTIFICATIONS_OLD_BASE_PATH/*/tokens").permitAll()
-            .requestMatchers(HttpMethod.GET, "$IDENTIFICATIONS_OLD_BASE_PATH/*/transaction-info").permitAll()
+            .requestMatchers("$IDENTIFICATIONS_BASE_PATH/*/tokens/*").permitAll()
+            .requestMatchers("$IDENTIFICATIONS_BASE_PATH/*/tokens").permitAll()
+            .requestMatchers(HttpMethod.GET, "$IDENTIFICATIONS_BASE_PATH/*/transaction-info").permitAll()
             .requestMatchers("$IDENTIFICATIONS_BASE_PATH/**").authenticated()
             .requestMatchers("$IDENTIFICATIONS_BASE_PATH/**").hasAuthority(MANAGE_IDENTIFICATION_SESSION_AUTHORITY)
-            .requestMatchers("$IDENTIFICATIONS_OLD_BASE_PATH/**").authenticated()
```

Comment:
We still need this, as long as the old endpoints are still available. Or am I missing something?! 🤔

Reply:
@JulianRoesner

Reply:
I just saw, that meanwhile the old endpoints are removed.. so nevermind